source/adapters/native_cpu/ur_interface_loader.cpp

```diff
@@ -292,6 +292,20 @@ UR_DLLEXPORT ur_result_t UR_APICALL urGetCommandBufferExpProcAddrTable(
   return retVal;
 }
 
+UR_DLLEXPORT ur_result_t UR_APICALL urGetEventExpProcAddrTable(
+    ur_api_version_t version, ///< [in] API version requested
+    ur_event_exp_dditable_t
+        *pDdiTable ///< [in,out] pointer to table of DDI function pointers
+) {
+  auto retVal = validateProcInputs(version, pDdiTable);
+  if (UR_RESULT_SUCCESS != retVal) {
```

Comment:
I am not sure of the style used in this adapter, can comment just about the CUDA and HIP ones, but same nit-pick comment wrt Yoda notation used for the comparison. Again, nothing blocking though.